doc/src/guides/solving/solve-polynomial.md

```diff
@@ -0,0 +1,515 @@
+# Solve (Find the Roots of) a Polynomial Algebraically or Numerically 
+
+Use SymPy to solve (find the roots of) a univariate polynomial algebraically.
+For example, solving $ax^2 + bx + c = 0$ for $x$ yields $x =
+\frac{-b\pm\sqrt{b^2 - 4ac}}{2a}$.
+
+## Alternatives to Consider
+
+- If you need a numeric (rather than algebraic) solution, you can use either
+    - NumPy's {external:func}`~numpy.roots`
+    - SciPy's {external:func}`~scipy.optimize.root`
+- If you need to solve systems of polynomial equations algebraically, use
+  {func}`~.solve`
+
+## Example of Finding the Roots of a Polynomial Algebraically
+
+Here is an example of solving a polynomial algebraically:
+
+```py
+>>> from sympy import roots
+>>> from sympy.abc import x, a, b, c
+>>> roots(a*x**2 + b*x + c, x)
+{-b/(2*a) - sqrt(-4*a*c + b**2)/(2*a): 1,
+ -b/(2*a) + sqrt(-4*a*c + b**2)/(2*a): 1}
+```
+
+This example reproduces the [quadratic
+formula](https://en.wikipedia.org/wiki/Quadratic_formula).
+
+## Functions to Find the Roots of a Polynomial
+
+There are several functions that you can use to find the roots of a polynomial:
+- {func}`~.solve` is a general solving function which can find roots, though is
```

Comment:
Maybe clarify that the advantage of `solve` is that it also works on non-polynomial equations (and cross-link to the other guide).